src/webgl/p5.RendererGL.Immediate.js

```diff
@@ -134,6 +134,12 @@ p5.RendererGL.prototype.vertex = function(x, y) {
         u /= this._tex.width;
         v /= this._tex.height;
       }
+    } if (
```

Comment:
Oh one more thing, should this be an else if?

```suggestion
    } else if (
```

Reply:
> Oh one more thing, should this be an else if?

okay cool, I thought they would do the name thing.